src/pages/VideoPage/VideoPage.jsx

```diff
@@ -153,6 +161,20 @@ const Video = () => {
             <span>|</span>
             <p>{categoryName}</p>
           </div>
+
+          <p className="video-section__info">
+            <span>
+              {" "}
+              Aired on{" "}
+              {new Date(createdAt).toLocaleDateString("en-US", {
+                weekday: "long",
+                year: "numeric",
+                month: "long",
+                day: "numeric",
+              })}
```

Comment:
* You can extract this out

Reply:
fixed - [9c439eb](https://github.com/SukanyaSen530/DrawFlix/pull/13/commits/9c439eba9ce6c798f6ff67915789c3938e1afc6d)